Tests/TestAppUITests/TestAppUITests.swift

```diff
@@ -23,6 +28,24 @@ class TestAppUITests: XCTestCase {
             XCTAssertEqual(error as? ScreenObject.WaitForScreenError, .timedOut)
         }
     }
+
+    func testScreenInitThrowsWhenScreenWithMultipleElementsIsNotLoaded() throws {
+        do {
+            _ = try MissingMultipleElementsScreen(app: app)
+            XCTFail("Expected `ScreenObject` `init` to throw, but it didn't")
+        } catch {
+            XCTAssertEqual(error as? ScreenObject.WaitForScreenError, .timedOut)
+        }
+    }
+
+    func testInitThrowsWhenGivenEmptyGettersArray() throws {
+        do {
+            _ = try ScreenObject(expectedElementGetters: [], app: app)
+            XCTFail("Expected `ScreenObject` `init` to throw, but it didn't")
+        } catch {
+            XCTAssertEqual(error as? ScreenObject.InitError, .emptyExpectedElementGettersArray)
```

Comment:
Same mechanism as the previous test, but this time to ensure that, if we pass an empty array, the initialization fails.

Some languages such as [Idris](https://github.com/idris-lang/Idris-dev/blob/9f5ceb7d16690efe648e07452f27306d2f5411d5/libs/prelude/Prelude/List.idr#L50-L57) have ways to define an input array has to be non empty (although I haven't fully grokked the way that works 😅 ) but Swift doesn't. There is one third-party library that adds a `NonEmpty<Array>` type, but it seemed too much overhead to bring it in just for this feature.